test/message/promise_reject.js

```diff
@@ -0,0 +1,22 @@
+// Flags: --expose-gc
+
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+
+Promise.reject(new Error('oops'));
+
+// Manually call GC due to possible memory constraints with attempting to
```

Comment:
Why is this inside a `setTimeout` with 2 delay? Can't the handlers be added prior and then the global.gc calls made synchronously? (Also, why 3 times)

Reply:
See above

Reply:
probably was arbitrary